1_processing/README.md

```diff
@@ -1,29 +1,31 @@
 # Processing
 The Raw data was quantified in Salmon using both PAO1 and PA14 references.
-For more information on the raw data is see this [paper](link TBD) with source code [here](https://github.com/hoganlab-dartmouth/pa-seq-compendia).
-The dataset containing all sample aligned against the PAO1 reference is [here](https://github.com/hoganlab-dartmouth/pa-seq-compendia/blob/main/qc_filtering/qc-out/pao1_aligned_compendium_p2_filtered_counts_norm.csv) and the against the PA14 reference is [here](https://github.com/hoganlab-dartmouth/pa-seq-compendia/blob/main/qc_filtering/qc-out/pa14_aligned_compendium_p2_filtered_counts_norm.csv)
+For more information on the raw data is see this [Doing et al.](https://www.biorxiv.org/content/10.1101/2022.01.24.477642v1) with source code [here](https://github.com/hoganlab-dartmouth/pa-seq-compendia).
+The dataset containing all strains aligned against the PAO1 reference and PA14 reference is [here](https://osf.io/vz42h/).
```

Comment:
```suggestion
The datasets containing all strains aligned against the PAO1 reference and PA14 reference is [here](https://osf.io/vz42h/).
```

I thought maybe this should be plural because its two matrices? but definitely your choice if you want to ignore this